packages/manager/apps/web/client/app/domain/options/delete/domain-options-delete.html

```diff
@@ -0,0 +1,17 @@
+<div data-ng-controller="controllers.Domain.Options.Delete as ctrl">
+    <oui-modal
+      data-heading="{{:: ('domain_tab_options_delete_'+ctrl.option.option+'_title') | translate }}"
+      data-loading="ctrl.loading"
+      data-on-dismiss="ctrl.closeModal()"
+      data-primary-action="ctrl.deleteDomain()"
+      data-primary-disabled="ctrl.loading"
+      data-primary-label="{{:: 'domain_tab_options_delete_confirm_button' | translate }}"
+      data-secondary-action="ctrl.closeModal()"
+      data-secondary-disabled="ctrl.loading"
+      data-secondary-label="{{:: 'domain_tab_options_delete_cancel' | translate }}">
+        <p data-translate="domain_tab_options_delete_confirm"></p>
+        <oui-message type="warning">
```

Comment:
```suggestion
        <oui-message data-type="warning">
```

Reply:
Not part of this PR. It's just a `git subtree pull` from the original `develop` branch (https://github.com/ovh-ux/ovh-manager-web).